### ISSUE TITLE
Firefox 13 will not start

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#7]
```
When attempting to start Firefox 13 nothing happens. Selenium has been updated 
and this issue has been resolved. I would try building this from source with 
the updated version of Selenium but I've got to leave for work.

From: http://selenium.googlecode.com/svn/trunk/dotnet/CHANGELOG

"
v2.23.0
=======
 * Removed the embedded IE server from the .NET bindings. From now on, you 
   *must* use the IE standalone server with .NET.
 * Removed unneeded polling for parent.lock file on anonymous Firefox profile
   in temp directory.
"

There are also some notes about the Internet Explorer Driver Server mentioned 
on http://seleniumhq.org/download/ I don't know if it will complicate things 
for you or not.
```

Original issue reported on code.google.com by `matthewkastor@gmail.com` on 11 Jun 2012 at 11:04

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.5.3 : 
 Updated Selenium libraries to 2.23.0
 Updated IEDriverServer to 2.23.0
 Updated Chrome Driver to 20.0.1133.0
Thanks for the issue report!

```

Original comment by `florentbr` on 11 Jun 2012 at 1:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Awesome! Thanks for updating. :D
```

Original comment by `matthewkastor@gmail.com` on 11 Jun 2012 at 1:40
